components/public-api-server/pkg/apiv1/identityprovider.go

```diff
@@ -93,7 +93,8 @@ func (srv *IdentityProviderService) GetIDToken(ctx context.Context, req *connect
 	userInfo.SetName(user.Name)
 	userInfo.AppendClaims("user_id", user.ID)
 	userInfo.AppendClaims("org_id", workspace.Workspace.OrganizationId)
-	userInfo.AppendClaims("context", workspace.Workspace.ContextURL)
+	// using Workspace.Context.NormalizedContextURL to not include prefixes (like "referrer:jetbrains-gateway", or other prefix contexts)
+	userInfo.AppendClaims("context", workspace.Workspace.Context.NormalizedContextURL)
```

Comment:
Code LGTM, will check if we always have `workspace.Workspace.Context.NormalizedContextURL` set